### DISCUSSION TITLE
rabbitmq-auth-backend-http plugin

### DISCUSSION THREAD

**Discussion by MohammadHatami20** [rabbitmq/rabbitmq-server#5389] [Other]
Can we support the authentication part with third-party API , but for the authorization part do it with Rabbitmq.
I mean when we want to check if the pass for the user is correct, do it whit third-party API, but for checking perminisson using internal Rabbitmq way.

**Comment by michaelklishin** [comment]
This is covered in [Alternative Authentication and Authorisation Backends](https://www.rabbitmq.com/access-control.html#backends).

**Comment by michaelklishin** [comment]
For instance, one of the examples from the above doc section uses:

``` ini
# rabbitmq.conf
#
auth_backends.1.authn = ldap
auth_backends.1.authz = internal
```
